packages/preview/typographix-polytechnique-reports/0.1.5/typst.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "typographix-polytechnique-reports"
+version = "0.1.5"
+entrypoint = "polytechnique.typ"
+authors = ["Rémi Germe <@remigerme>"]
+license = "MIT"
+description = "A report template for Polytechnique students (from TypographiX)."
+repository = "https://github.com/remigerme/typst-polytechnique"
+keywords = ["polytechnique", "école polytechnique", "report", "rapport", "stage", "typographix"]
+categories = ["report"]
+exclude = [
+    ".gitignore",
+    "guide.typ", "guide.pdf",
+    "assets/example.bib", "assets/typst.png",
```

Comment:
[This test](https://github.com/typst/packages/pull/1427/checks?check_run_id=34417762953) fails because `assets/example.bib` should be part of the exclude section. It seems it is indeed the case. What am I missing ?

Reply:
No worries, the CI is just reporting a false positive here, sorry about that.